Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -284,8 +299,8 @@ - (void)urlSessionTask:(NSURLSessionTask *)sessionTask setState:(NSURLSessionTas
             NSNumber *statusCode = [NSNumber numberWithInteger:responseStatusCode];
 
             if (netSpan != nil) {
-                [netSpan setTagValue:[NSString stringWithFormat:@"%@", statusCode]
-                              forKey:@"http.status_code"];
+                [netSpan setDataValue:[NSString stringWithFormat:@"%@", statusCode]
+                              forKey:@"http.response.status_code"];
```

Comment:
This change was made in response to https://github.com/getsentry/team-mobile/issues/121